src/components/performances/PerformanceTableRow.jsx

```diff
@@ -12,77 +12,44 @@ class PerformanceTableRow extends React.Component {
   render() {
     const {
       id,
-      entryId,
-      collectionName,
-      danceTitle,
-      school,
-      academicLevel,
-      competitionLevel,
-      danceStyle,
-      performers,
-      choreographers,
-      groupSize,
+      currentValues,
       eventId
     } = this.props;
-    const currentValues = {
+    const {
+      danceEntry,
```

Comment:
not sure about this naming, it used to be an id, now its an entry. Is this the Dance Entry value (0, 1, ...)?

Reply:
@YangStephen `danceEntry` is the entry id (0,1,2,...) for a specific performance, where as the `id` is the document id for the performance (a unique identifier we use for the sepcific docemnt in the performances collection in firestore i.e. `WH9oJH8PVT2coTECdTSG`), both values are still there, we just pass the id as a prop and the danceEntry inside the `currentValues` object